src/test/java/com/infogain/debonair/serviceTest/CustomerServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.infogain.debonair.serviceTest;
+
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.infogain.debonair.repository.RegularAndCorporateCustomer;
+import com.infogain.debonair.service.CustomerService;
+
+@ExtendWith(MockitoExtension.class)
+public class CustomerServiceTest {
+	
+	
+	@Mock
+	RegularAndCorporateCustomer regularAndCorporateCustomerRepo;
+	
+	@InjectMocks
+	CustomerService customerService;
+	
+	@BeforeAll
+	public static void beforeClass(){
+        System.out.println("This is to show use of @BeforeAll annotation in CustomerServiceTest.");
+        System.out.println("This will be executed before all the test cases.");
+    }
```

Comment:
what is the use here???

Reply:
This was just to test the functionality for my own learning. It has nothing to do with the project.